runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -336,4 +348,46 @@ private static String toUrgency(TrayIcon.MessageType type)
 				return "normal";
 		}
 	}
+
+	private void playCustomSound()
+	{
+		Clip clip = null;
+
+		// Try to load the user sound from ~/.runelite/notification.wav
+		File file = new File(RuneLite.RUNELITE_DIR, "notification.wav");
+		if (file.exists())
+		{
+			try
+			{
+				InputStream fileStream = new BufferedInputStream(new FileInputStream(file));
+				try (AudioInputStream sound = AudioSystem.getAudioInputStream(fileStream))
```

Comment:
I feel like the sound should be pre-loaded on config change, this will be running in game thread

Reply:
I've opted to just put this in the executor, because I don't want to have to deal with support for people's custom sounds not loading.